lib/client/services/creative/creative.type.ts

```diff
@@ -255,7 +255,7 @@ export type CreativeAsset = {
    *
    * The assetByteArray will be null when the creative is retrieved.
    */
-  assetByteArray: BinaryData;
+  assetByteArray: string;
```

Comment:
Don't we need some sort of a type for base64 instead of a string or that decoding/encoding part could be done later/downstream when this `CreativeAsset` type is being used?